test/e2e/util.go

```diff
@@ -1156,6 +1160,10 @@ func RunRC(config RCConfig) error {
 			Diff(oldPods, pods).Print(util.NewStringSet())
 			return fmt.Errorf(errorStr)
 		}
+		//TODO: Convert this warning to failure after #10030 is fixed.
+		if len(pods) > config.Replicas {
+			LogfPrefix("WARN", "POD overshoot occured %d (actual pods) > %d (replicas) !!! See https://github.com/GoogleCloudPlatform/kubernetes/issues/10030. This is not necessarily a failure, but hopefully it will be repaired soon.", len(pods), config.Replicas)
```

Comment:
We could but then it would be "INFO: WARNING..." From what I can tell because of the hardcoded info... Am I missing something?